### DISCUSSION TITLE
video "loop" button - feasibility?

### DISCUSSION THREAD

**Discussion by ElectronicBattle** [blakeblackshear/frigate#13180] [Ideas]
Hello all

For video recordings, could a "loop once/loop permanently" toggle be provided in the UI or via the config please?

When looking for wildlife (even garden birds etc.) it can be useful to replay numerous times a video clip at 16x rate in order to spot changes. 
It's difficult to accurately describe the situation, but you might imagine a deer looking at you through long grass and then withdrawing and moving slowly, carefully away. Difficult to spot at 1x speed, but if you play it in a continuous loop at full speed, the differences become apparent.

**Comment by hawkeye217** [comment]
It's not likely that Frigate will ever have a specific loop button, but the easiest way to achieve the view you're looking for would be to export the footage from Frigate and then use ffmpeg or your computer's media player software to speed it up, scrub, loop, etc.

**Reply by ElectronicBattle** [reply]
True - I'm currently using mplayer for just that reason.
